modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/dto/v1_0/converter/AccountResourceDTOConverter.java

```diff
@@ -83,17 +87,30 @@ public Account toDTO(
 
 		return new Account() {
 			{
+				accountIds = TransformUtil.transformToArray(
```

Comment:
What is this field?

Reply:
@luca-pellizzon , replying before I go to sleep. We can remove the field from being displayed in the DTO, but we need accountIds (See: https://github.com/liferay-commerce/liferay-portal/pull/3838/commits/ef1180058a1ad46ea40838412ee96c2833a502de and line 375 (https://github.com/liferay-commerce/liferay-portal/pull/3838/commits/b3db47336122be73c7650ac88ef09b39004ebe35#diff-a4ad6144078aa8accc7d7e0355f20aa65a7ee84d675a5b88670f43ecbc220b3f) to do update logic for accountEntryUserRel (this is the equivalent of AccountMember update).

Reply:
ok, but this is still wrong
`AccountEntryUserRel::getAccountEntryId`
should be
`AccountEntryUserRel::getAccountUserId`
or we get a list of the same ID (the account entry ID) repeated `n` times, where `n` is the number of users linked to the account, right?